src/components/component/items/MypageCard.jsx

```
@@ -0,0 +1,44 @@
+import { FaRegTrashAlt } from "react-icons/fa";
+
+const MypageCard = ({ review, handleCourseClick }) => {
+        return (
+            <div className='review-card'>
+                <div className="myreview-content">
+                    <div className='myreview-top'>
+                        <span
+                            className='myreview-title'
+                            onClick={() => handleCourseClick(review)}
+                            style={{ cursor: 'pointer' }}
+                        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)